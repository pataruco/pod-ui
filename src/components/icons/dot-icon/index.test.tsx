import { shallow } from 'enzyme';
import React from 'react';
import DotIcon from './index';

describe('DotIcon', () => {
  it('renders', () => {
    expect(shallow(<DotIcon />)).toMatchSnapshot();
  });
});
