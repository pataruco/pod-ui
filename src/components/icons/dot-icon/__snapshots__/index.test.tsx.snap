// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DotIcon renders 1`] = `
<svg
  className="svg"
  viewBox="0 0 52 52"
>
  <circle
    cx="26"
    cy="26"
    r="24"
  />
</svg>
`;
